index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <h1 id="title">Our <strong>Survey</strong> Form</h1>
+
+      <!-- <section id="hero">
+        <div class="hero container">
+          <div>
+            <h1>Hello, <span></span></h1>
+            <h1>My Name is <span></span></h1>
+            <h1>Hamza <span></span></h1>
+            <a href="#" type="button" class="cta">Portfolio</a>
+          </div>
+        </div>
+      </section> -->
+
+      <p id="description">
+        You want to know more about our coffee? Send us a message.
+      </p>
+      <form id="survey">
+        <label for="survey-form">Name</label>
+        <input
+          type="text"
+          id="survey-form"
+          name="name-label"
+          placeholder="Name"
+        />
+
+
+
+
+
```

Comment:
Please, remove the blank lines.

Reply:
- [ ] as I mentioned in the previous comments your comment should be more polite. So it should also be a suggestion instead of making it look like an order.  For example, you can say, "**It will be better** to avoid blank lines in your code. They will make your project look messy and to long".